src/core/reader/types/values.rs

```diff
@@ -95,6 +95,29 @@ impl WasmReader<'_> {
         let len = self.read_var_u32()?;
         (0..len).map(|_| read_element(self)).collect()
     }
+
+    pub fn read_instruction(&mut self) -> Result<&[u8]> {
+        if self.current.is_empty() {
+            return Err(Error::Eof);
+        }
+        match self.current {
+            // check if we are at a multibyte (2 byte) instruction
+            [0xFB, _, ..] | [0xFC, _, ..] | [0xFD, _, ..] | [0xFE, _, ..] => {
```

Comment:
Why don't we use the consts here?

Reply:
You are correct, will fix them to use consts here

Reply:
Used them in both executor and validator. Please check: 
- [executor](https://github.com/DLR-FT/wasm-interpreter/blob/efa6eac09f56d6ce20aa1ef6efe4198b870522b8/src/execution/mod.rs)
- [validator](https://github.com/DLR-FT/wasm-interpreter/blob/efa6eac09f56d6ce20aa1ef6efe4198b870522b8/src/validation/code.rs)